src/pages/Search/SearchResults/SearchResults.test.tsx

```diff
@@ -1,77 +1,93 @@
-import { render } from '@testing-library/react';
```

Comment:
Flytt testene til Search.tsx for bedre integrasjonstesting (les begrunnelse i kommentar til SearchArea.test-filen)